packages/scaffold/src/views/w3m-onramp-activity-view/index.ts

```diff
@@ -104,6 +106,33 @@ export class W3mOnRampActivityView extends LitElement {
 
     this.loading = false
     this.coinbaseTransactions = coinbaseResponse.data || []
+    this.refetchLoadingTransactions()
+  }
+
+  private refetchLoadingTransactions() {
+    const loadingTransactions = this.coinbaseTransactions.filter(
+      transaction => transaction.metadata.status === 'ONRAMP_TRANSACTION_STATUS_IN_PROGRESS'
+    )
+
+    if (loadingTransactions.length === 0) {
+      clearTimeout(this.refetchTimeout)
+
+      return
+    }
+
+    // Wait 2 seconds before refetching
+    this.refetchTimeout = setTimeout(async () => {
```

Comment:
This looks like it should have https://developer.mozilla.org/en-US/docs/Web/API/AbortController implemented to cancel previous request?